### DISCUSSION TITLE
Strange huge virtual frequency

### DISCUSSION THREAD

**Discussion by zhaowb12138** [SSCHAcode/python-sscha#182] [Q&A]
Dear developers,
Recently, I encountered an issue while using SSCHA.The V3-H3SS matrix shows an exceedingly large imaginary frequency, exceeding 1500 cm⁻¹. I am uncertain whether this outcome suggests that my structure remains unstable even after accounting for the anharmonic effect or if there might be issues with the software I am using.
This is the harmonic phonon spectrum
![image](https://github.com/SSCHAcode/python-sscha/assets/108879161/f507fd78-623f-42bc-b180-178225b59360)
This is the convergence result of force constant gradient with ensembles have 200 , 500 and 1000 configurations
![image](https://github.com/SSCHAcode/python-sscha/assets/108879161/744e7a92-8109-4365-8afc-7876473fe06c)
This is V3-HESS phonon spectrum
![image](https://github.com/SSCHAcode/python-sscha/assets/108879161/360c0dd4-2083-4123-a915-75c69e853ec0)
Thanks a lot!
Wenbo Zhao

**Comment by mesonepigreco** [comment]
Hi, 
Sorry for the late reply!
Your minimization has converged. It really seems that the phonons remain unstable here. However, indeed, the plot is strange, as it seems that you have an imaginary frequency at gamma that was not present at the harmonic level.
If you are strongly unstable, it is possible to have a different instability with the SSCHA. 
Have you used the last dynamical matrix to compute the free energy Hessian? Can you share the specific code you used for it? Maybe we can  help identifying some problems.
